src/assets/styles/theme.scss

```diff
@@ -27,7 +27,7 @@ $BCgovABlue1: #6e93ff;
 $BCgovABlue2: #3b6cff;
 $BCgovABlue3: #0846ff;
 $BCgovABlue4: #003bee;
-$primary-blue: #1976d2;
+$primary-blue: #1976d2; // used by StaffComment
```

Comment:
I think StaffComment should actually use `$app-blue`, but that's not in scope for this ticket.

Reply:
I updated the shared component to use `$app-blue` instead, reimported it here, and deleted this obsolete colour.